app/helpers/dashboard_helper.rb

```diff
@@ -16,6 +16,10 @@ def total_received_money_donations(range = selected_range)
     current_organization.donations.during(range).sum { |d| d.money_raised || 0 }
   end
 
+  def total_received_money_donations_from_product_drives(range = selected_range)
```

Comment:
```suggestion
  def total_received_money_donations_from_product_drives(range: selected_range)
```
I have a preference for using keyword arguments so it is clear when it is being used what the parameter is.